golem-worker-executor-base/tests/common/mod.rs

```diff
@@ -305,6 +305,11 @@ pub async fn start(context: &TestContext) -> anyhow::Result<TestWorkerExecutor>
             port: context.grpc_port(),
             access_token: "03494299-B515-4427-8C37-4C1C915679B7".to_string(),
         },
+        // memory: MemoryConfig {
```

Comment:
is this needed or todo?

Reply:
It makes it possible to test limited memory scenarios I left it there intentionally for the next step. In that I'm planning to have dedicated test cases doing this and remove the commented out lines here